src/main/resources/interface-specification.yml

```diff
@@ -9795,12 +9771,18 @@ components:
       title: AttributesResponse
       type: object
       properties:
-        attributes:
+        totalCount:
+          type: integer
+        limit:
+          type: integer
+        offset:
+          type: integer
```

Comment:
these need to be replaced with `Pagination` (see other listing in this file for reference)